Add Sig-Usability-Spotlight Blog (#27809)
kubernetes/website: content/en/blog/_posts/2021-04-29-SIG-Usability-Spotlight.md

SIG Usability focuses on new users and existing clients, evaluating their feedback to improve user experience.

### Q. Why should new and existing contributors consider joining SIG Usability?

A. There are plenty of territories where new supporters can begin with. User research project where people can help in understanding the usability of the end-user facing experiences, be that error messages, end-to-end tasks, etc. 

Accessibility guidelines for Kubernetes community artifacts, examples include internationalization of documentations, color choices for people with color blindness, ensuring compatibility with screen reader technology, user interface design for core components with user interfaces, and more.

### Q. What do you do to help new contributors get started?

A. New contributors can get started by shadowing one of the user interviews, going through user interview transcripts, analyzing user interviews, and designing surveys.

> SIG Usability is also open to new project ideas. If you have an idea, we’ll do what we can to fulfill that idea. There are also regular SIG Meetings for people to attend and ask their questions live. The meetings are also recorded for those who may not be able to attend. As always, you can reach out to us on Slack as well.

Reviewer: ```suggestion
SIG Usability is also open to new project ideas. If you have an idea, we’ll do what we can to support it. There are regular SIG Meetings where people can ask their questions live. These meetings are also recorded for those who may not be able to attend. As always, you can reach out to us on Slack as well.
```

A. In simple terms, the survey gathers information about how people use Kubernetes, such as trends in learning to deploy a new system, error messages they receive, and workflows.

One of our goals is to standardise the responses accordingly. The ultimate goal is to analyze survey responses for important user stories whose needs aren't being met.

### Q. Are there any particular skills you’d like to recruit for? What skills are contributors to SIG Usability likely to learn?

A. Although contributing to SIG Usability does not have any pre-requisites as such, but experience with user research, qualitative research, prior experience with how to conduct an interview would be a great plus point. Quantitative research, like survey design and screening, is also helpful and something that we expect contributors to learn.

### Q. What are you getting positive feedback on, and what’s coming up next for SIG Usability?

A. We have had new members joining and coming to monthly meetings regularly and showing interests in becoming a contributor and helping the community. We have also had a lot of people reach out to us via Slack showcasing their interest in the sig. 

> Currently, we are focused on finishing this study, which is our project for this year. We are always happy to have new contributors join us.

Reviewer: This is the first mention of the study, is there any other information on it?

### Q: Any closing thoughts/resources you’d like to share?

> A. We love meeting new contributors and perceive how we can assist them with investigating different spaces of the Kubernetes project. We will work with and team up with other SIGs on giving end-user channels for demonstrating input on their experience, studies, and other criticism get-togethers around project convenience, documentation, plan components, and language rules for Kubernetes concepts, recognizing opportunities for user research and best strategies to accomplish those.

Reviewer: ```suggestion
A. We love meeting new contributors and assisting them in investigating different Kubernetes project spaces. We will work with and team up with other SIGs to facilitate engaging with end-users, running studies, and help them integrate accessible design practices into their development practices.
```

The 2nd sentence didn't make much sense as read, I think this suggestion covers the intent of it? :x

---
layout: blog
title: "SIG-Usability Spotlight"
date: 2021-04-29
slug: sig-usability-spotlight-2021
---

> **Authors:** [Kunal Kushwaha](https://twitter.com/kunalstwt)

Reviewer 2: If it's just you, no need to pluralize `**Author:**`

> Are you interested in learning about what Sig-Usability does and how you can get involved? Well, you're at the right place. Sig-Usability is all about making Kubernetes more accessible to new folks and the main activity includes conducting user research for the community. In this blog, we have summarised our conversation with Gaby Moreno who walks us through the various aspects of being a part of the SIG and sharing with us some insights about how others can get involved. 

Reviewer 2: ```suggestion
Are you interested in learning about what Sig-Usability does and how you can get involved? Well, you're at the right place. Sig-Usability is all about making Kubernetes more accessible to new folks, and its main activity is conducting user research for the community. In this blog, we have summarized our conversation with Gaby Moreno who walks us through the various aspects of being a part of the SIG and shares some insights about how others can get involved.
```

### Q. What are you getting positive feedback on, and what’s coming up next for SIG-usability?

a. We have had new members joining and coming to monthly meetings regularly and showing interests in becoming a contributor and helping the community. We have also had a lot of people reach out to us via Slack showcasing their interest in the sig. 

Currently, we are focused on finishing this study which is our project for this year and we are always happy to have new contributors join us.

### Q: Any closing thoughts/resources you’d like to share?

A. We love meeting new contributors and perceive how we can assist them with investigating different spaces of the Kubernetes project. We will work with and team up with other SIGs on giving end-user channels for demonstrating input on their experience, studies and other criticism get-togethers around project convenience, documentation, plan components and language rules for Kubernetes concepts, recognizing opportunities for user research and best strategies to accomplish those.

Here are some resources for you to get started:

> - [GitHub](https://docs.google.com/document/d/17zSSDRYoVySbxev2_9r1x89YzFtRH-5DUrz-Q3VAGuo/edit)

Reviewer 2: Is this the correct link (or link text)?

> - [GitHub](https://docs.google.com/document/d/17zSSDRYoVySbxev2_9r1x89YzFtRH-5DUrz-Q3VAGuo/edit)

Author: Oops my bad

- [Mailing list](https://groups.google.com/g/kubernetes-sig-usability)
- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fusability)

> - Slack: #sig-usability

Reviewer 2: You can link to the slack channel as well: https://kubernetes.slack.com/archives/CLC5EF63T

---
layout: blog

> title: "SIG-Usability Spotlight"

Reviewer 3: How about the following title:

# Spotlight on SIG Usability

date: 2021-04-29
slug: sig-usability-spotlight-2021
---

**Author:** [Kunal Kushwaha](https://twitter.com/kunalstwt)

## Introduction

> Are you interested in learning about what Sig-Usability does and how you can get involved? Well, you're at the right place. Sig-Usability is all about making Kubernetes more accessible to new folks, and its main activity is conducting user research for the community. In this blog, we have summarized our conversation with [Gaby Moreno](https://twitter.com/morengab) who walks us through the various aspects of being a part of the SIG and shares some insights about how others can get involved.

Reviewer 3: It might be good to have some consistency in naming "SIG Usability".

E.g., "Sig-Usability" is used in this paragraph, and "SIG Usability" in the next.

Reviewer 3: And "SIG-Usability" further below.

> Are you interested in learning about what Sig-Usability does and how you can get involved? Well, you're at the right place. Sig-Usability is all about making Kubernetes more accessible to new folks, and its main activity is conducting user research for the community. In this blog, we have summarized our conversation with [Gaby Moreno](https://twitter.com/morengab) who walks us through the various aspects of being a part of the SIG and shares some insights about how others can get involved.

Reviewer 3: Also, is there a resource you can link the first occurrence of "SIG Usability" to?

Author: Thanks for the comments, changes made

> Are you interested in learning about what [SIG-Usability](https://github.com/kubernetes/community/tree/master/sig-usability) does and how you can get involved? Well, you're at the right place. SIG-Usability is all about making Kubernetes more accessible to new folks, and its main activity is conducting user research for the community. In this blog, we have summarized our conversation with [Gaby Moreno](https://twitter.com/morengab) who walks us through the various aspects of being a part of the SIG and shares some insights about how others can get involved.

Reviewer 4: ```suggestion
Are you interested in learning about what [SIG-Usability](https://github.com/kubernetes/community/tree/master/sig-usability) does and how you can get involved? Well, you're at the right place. SIG-Usability is all about making Kubernetes more accessible to new folks, and its main activity is conducting user research for the community. In this blog, we have summarized our conversation with [Gaby Moreno](https://twitter.com/morengab), who walks us through the various aspects of being a part of the SIG and shares some insights about how others can get involved.
```

> Gaby is a Co-Lead for SIG Usability. She works as a Product Designer at IBM and enjoys working on the user experience of open, hybrid cloud technologies like Kubernetes, OpenShift, Terraform, and Cloud Foundry.

Reviewer 3: Lowercase "co-lead"

> ## A brief summary of our converstation is captured below

Reviewer 4: s/converstation/conversation/

> ## A brief summary of our converstation is captured below

Reviewer 4: ```suggestion
## A summary of our conversation is captured below
```

### Q. Could you tell us a little about what SIG-Usability does?

> A. SIG-Usability at a high level started because there was no dedicated user experience for Kubernetes. The extent of SIG convenience is the centre of end-client ease of use of the Kubernetes project. The main activity is user research for the community. Which includes speaking to people that use Kubernetes. 

Reviewer 2: Please use [US spelling](https://kubernetes.io/docs/contribute/style/style-guide/#language): "`center`"

> A. SIG-Usability at a high level started because there was no dedicated user experience for Kubernetes. The extent of SIG convenience is the centre of end-client ease of use of the Kubernetes project. The main activity is user research for the community. Which includes speaking to people that use Kubernetes. 

Reviewer 2: "`center`"

> A. SIG-Usability at a high level started because there was no dedicated user experience for Kubernetes. The extent of SIG convenience is the centre of end-client ease of use of the Kubernetes project. The main activity is user research for the community. Which includes speaking to people that use Kubernetes. 

Reviewer 4: ```suggestion
A. SIG-Usability at a high level started because there was no dedicated user experience for Kubernetes. The extent of SIG convenience is the centre of end-client ease of use of the Kubernetes project. The main activity is user research for the community, which includes speaking to Kubernetes users. 
```

> This covers points like client experience and availability. The objectives of the SIG are to guarantee that the Kubernetes project is maximally usable by clients of a wide range of foundations and capacities. Instances of endeavours incorporate internationalization and openness of documentation.

Reviewer 2: "`endeavors`"

> As you can see SIG-Usability cares about new users and existing clients by focussing on evaluating their feedback for user experience.

Reviewer 4: ```suggestion
SIG-Usability focuses on new users and existing clients, evaluating their feedback to improve user experience.
```

> ### Q. Why should new, or existing, contributors consider joining SIG-Usability?

Reviewer 4: ```suggestion
### Q. Why should new and existing contributors consider joining SIG-Usability?
```

A. There are plenty of territories where new supporters can begin with. User research project where people can help in understanding the usability of the end-user facing experiences, be that error messages, end-to-end tasks, etc. 

> Accessibility guidelines for Kubernetes community artifacts, examples include internationalization of documentations, colour choices for people with colour blindness, ensuring compatibility with screen reader technology, user interface design for core components with user interfaces, and more.

Reviewer 2: "`color`"

### Q. What do you do to help new contributors get started?

> A. New contributors can get started by shadowing one of the user interviews. Going through user interview transcripts and making analysis on user interviews. There are additionally plenty of opportunities in survey design background. 

Reviewer 4: ```suggestion
A. New contributors can get started by shadowing one of the user interviews, going through user interview transcripts, and analyzing user interviews. There are also opportunities in survey design background. 
```

SIG-Usability is also open to new project ideas. If you have an idea, we’ll do what we can to fulfil that idea. There are also regular SIG Meetings for people to attend and ask their questions live. The meetings are also recorded for those who may not be able to attend. As always, you can reach out to us on Slack as well.

### Q. What does the survey include?

> A. In simple terms, the survey includes information related to what are people doing with Kubernetes. We are looking for behaviours amongst users, examples include the trends of learning a new deployment system, error messages faced by the users, end-to-end tasks, etc. 

Reviewer 2: "`behaviors`"

> A. In simple terms, the survey includes information related to what are people doing with Kubernetes. We are looking for behaviours amongst users, examples include the trends of learning a new deployment system, error messages faced by the users, end-to-end tasks, etc. 

Reviewer 4: ```suggestion
A. In simple terms, the survey includes information related to what people are doing with Kubernetes. We are looking for behaviours amongst users, examples include the trends of learning a new deployment system, error messages faced by the users, end-to-end tasks, etc. 
```

> One of our goals is to standardise the responses accordingly. The ultimate goal being analysis the survey responses and looking for under-served jobs, areas with high importance but least satisfaction.

Reviewer 2: "`standardize`"

> One of our goals is to standardise the responses accordingly. The ultimate goal being analysis the survey responses and looking for under-served jobs, areas with high importance but least satisfaction.

Reviewer 4: ```suggestion
One of our goals is to standardise the responses accordingly. The ultimate goal is to analyze survey responses for important user stories whose needs aren't being met.
```

### Q. Are there any particular skills you’d like to recruit for? What skills are contributors to SIG-Usability likely to learn?

> A. Although contributing to SIG-Usability does not have any pre-requisites as such, but experience with user research, qualitative research, prior experience with how to conduct an interview would be a great plus point. Along with that, quantitative research, like survey design, screening etc. are also helpful and some things that we expect the contributors to learn.

Reviewer 4: ```suggestion
A. Although contributing to SIG-Usability does not have any pre-requisites as such, but experience with user research, qualitative research, prior experience with how to conduct an interview would be a great plus point. Quantitative research, like survey design and screening, is also helpful and something that we expect contributors to learn.
```

### Q. What are you getting positive feedback on, and what’s coming up next for SIG-Usability?

a. We have had new members joining and coming to monthly meetings regularly and showing interests in becoming a contributor and helping the community. We have also had a lot of people reach out to us via Slack showcasing their interest in the sig. 

Currently, we are focused on finishing this study which is our project for this year and we are always happy to have new contributors join us.

### Q: Any closing thoughts/resources you’d like to share?

> A. We love meeting new contributors and perceive how we can assist them with investigating different spaces of the Kubernetes project. We will work with and team up with other SIGs on giving end-user channels for demonstrating input on their experience, studies and other criticism get-togethers around project convenience, documentation, plan components and language rules for Kubernetes concepts, recognizing opportunities for user research and best strategies to accomplish those.

Reviewer 4: ```suggestion
A. We love meeting new contributors and perceive how we can assist them with investigating different spaces of the Kubernetes project. We will work with and team up with other SIGs on giving end-user channels for demonstrating input on their experience, studies, and other criticism get-togethers around project convenience, documentation, plan components, and language rules for Kubernetes concepts, recognizing opportunities for user research and best strategies to accomplish those.
```

Here are some resources for you to get started:
- [GitHub](https://github.com/kubernetes/community/tree/master/sig-usability)
- [Mailing list](https://groups.google.com/g/kubernetes-sig-usability)
- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fusability)
- [Slack: #sig-usability](https://kubernetes.slack.com/archives/CLC5EF63T)

## Wrap Up

> SIG-Usability is hosting a [KubeCon talk](https://kccnceu2021.sched.com/event/iE6a/studying-kubernetes-users-with-sig-usability-gaby-moreno-cesar-ibm-josie-pynadath-tasha-drew-vmware-carl-j-pearson-userzoom) about studying Kubernetes' users experiences. The talk is going to be focussed on getting updates on the user study projects, to better understand who is using Kubernetes, what they are attempting to achieve, and how the project is addressing their necessities, and where we need to improve the project and the client experience. Join the SIG's update to find out about the most recent research results, what the plans are for the forthcoming year, and how to get involved in the upstream usability team as a contributor!

Reviewer 4: ```suggestion
SIG-Usability is hosting a [KubeCon talk](https://kccnceu2021.sched.com/event/iE6a/studying-kubernetes-users-with-sig-usability-gaby-moreno-cesar-ibm-josie-pynadath-tasha-drew-vmware-carl-j-pearson-userzoom) about studying Kubernetes users' experiences. The talk is going to be focussed on getting updates on the user study projects, to better understand who is using Kubernetes, what they are attempting to achieve, and how the project is addressing their necessities, and where we need to improve the project and the client experience. Join the SIG's update to find out about the most recent research results, what the plans are for the forthcoming year, and how to get involved in the upstream usability team as a contributor!
```
